lib/api/unittest/CDataFrameAnalyzerTest.cc

```diff
@@ -1018,6 +1021,66 @@ void CDataFrameAnalyzerTest::testCategoricalFields() {
     }
 }
 
+void CDataFrameAnalyzerTest::testCategoricalFields_EmptyAsMissing() {
+
+    auto eq = [](double expected) {
+        return [expected](double actual) { return expected == actual; };
+    };
+
+    auto nan = []() { return [](double actual) { return isnan(actual); }; };
```

Comment:
Can we make this maths::CDataFrameUtils::isMissing? This should match whatever value we use to indicate missing.
```suggestion
    auto missing = []() { return [](double actual) { return maths::CDataFrameUtils::isMissing(actual); }; };
```

Reply:
Done.